phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -1051,6 +1051,11 @@ public function format_methodparam($open, $name, $attrs) {
                 }
                 $content .= ' <span class="methodparam">';
                 ++$this->params["count"];
+                if (isset($attrs[Reader::XMLNS_DOCBOOK]["rep"]) && $attrs[Reader::XMLNS_DOCBOOK]["rep"] == "repeat") {
+                    $this->params["ellipsis"] = '...';
+                } else {
+                    $this->params["ellipsis"] = '';
+                }
```

Comment:
Why do we set `params["ellipsis"]` here rather than handling it directly in `format_methodparam_parameter`, like the `role` is?

Reply:
`role="reference"` is an attribute of `<parameter>`, but `rep="repeat"` is an attribute of `<methodparam>`.

Reply:
Oh, duh.